Remotion/Web/ClientScript/Scripts/ViewLayout.ts

```diff
@@ -16,47 +16,53 @@
 // 
 class ViewLayout
 {
-  public static AdjustActiveViewContent (viewContent: JQuery): void
+  public static AdjustActiveViewContent (viewContent: HTMLElement): void
   {
-    var viewContentBorder = viewContent.children('div').eq (0);
-    var viewBottomControls = viewContent.next();
+    var viewContentBorder = viewContent.querySelector<HTMLDivElement> (':scope > div')!;
+    var viewBottomControls = viewContent.nextElementSibling as HTMLElement;
 
-    var viewContentBorderHeight = viewContentBorder.outerHeight (true) - viewContentBorder.height();
-    var viewBottomControlsBorderHeight = viewBottomControls.outerHeight (true) - viewBottomControls.height();
+    var viewContentBorderHeight = LayoutUtility.outerHeight (viewContentBorder) - viewContentBorder.clientHeight;
+    var viewBottomControlsBorderHeight = LayoutUtility.outerHeight (viewBottomControls) - viewBottomControls.clientHeight;
```

Comment:
I have made the mistake of thinking `clientHeight` is the correct equivalent to jQuery's `.height()`, which it isn't. As such we need to create helpers for it.

Please create a commit adding those two methods to the `LayoutHelper` and use them instead of `.clientHeight`. Depending on which PR is merged first (yours or Stefans) the commit will be empty on rebase, since the other PR already contained the same code.

```ts
public static GetHeight(element: HTMLElement)
{
  // TODO RM-7734: Fix the width/height calculations in the LayoutHelper to respect box-sizing
  const style = window.getComputedStyle(element);
  return parseFloat(style.height) || 0;
}

public static GetWidth(element: HTMLElement)
{
  // TODO RM-7734: Fix the width/height calculations in the LayoutHelper to respect box-sizing
  const style = window.getComputedStyle(element);
  return parseFloat(style.height) || 0;
}
```

Reply:
Those methods should start with a lowercase letter right?

Reply:
+ the `get` prefix could be dropped, have a look at the other 2 methods in LayoutUtility.

Reply:
Actually, the other methods should start with an uppercase since all our methods are uppercase. But that got through the review. @MichaelKetting do you want `GetWidth` or `Width` as method name?

Reply:
Update: The layout helper methods are done in RM-7735. Once #63 is merged, rebase onto it to make use of the new `LayoutHelper` methods.

Reply:
#63 is merged :)